cmd/gateway/b2/gateway-b2_test.go

```diff
@@ -0,0 +1,119 @@
+/*
+ * Minio Cloud Storage, (C) 2017 Minio, Inc.
```

Comment:
We should add 2019 here.

Reply:
This is a revert not changing anything so will not add 2019